source/compliance/tsc/common-criteria/cc6.1.rst

```diff
@@ -39,7 +39,7 @@ Wazuh meets the architecture, infrastructure, and security software aspects of t
          </synchronization>
        </wodle>
 
-#. Navigate to **Agents management** > **Summary** > **AGENT_NAME** > **Inventory data** on the Wazuh dashboard. Where **AGENT_NAME** is the monitored endpoint. You can see details about installed applications, network services, and used ports on the monitored endpoint.
+#. Navigate to **Agents management** > **Summary** > **AGENT_NAME** > **Inventory data** on the Wazuh dashboard. Where **AGENT_NAME** is the monitored endpoint. You can see details about installed applications, network services, and used ports on the monitored endpoint. Click **Software**, **Network**, and **Processes** to see details about the installed software, network services, and running processes on the endpoint.
```

Comment:
**nitpick**: there are 2 sentences explaining the same thing. You could remove one of them or combine the information if you want to do some reference to the tab names.